### ISSUE TITLE
Expand and clear-up syntax: derive newtype instance

### ISSUE THREAD

**Issue by razcore-rad** [JordanMartinez/purescript-jordans-reference#474]
```
Section:
  Syntax

Issue Type:
  Enhancement

Folder/File(s):
  - Syntax/Basic-Syntax/src/TypeClasses-and-Newtypes/Deriving-Typeclass-Instances-for-Newtyped-Types.purs
```

I think `derive newtype instance` could use some more expansion in comparison with `derive instance`. I had to go to the original posts by paf31 to figure out that you can use `derive newtype instance` for any instance the wrapped type already defines, not just the regular: `Eq`, `Ord` & `Functor`.

I think it's enough just to mention that `derive newtype instance` forwards the instance from the wrapped type as long as the wrapped type defines it.

I can make a PR for this.

**Comment by JordanMartinez** [comment]
Good point. That could be better documented.

If you want to make a PR, you should target the `development` branch. Once PS 0.14 gets released, I'll merge that branch into master and make a release.

**Comment by razcore-rad** [comment]
Closed by #475
